_data/chains/eip155-868.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "Fantasia Chain Mainnet",
+  "chain": "FSC",
+  "rpc": [
+    "https://mainnet-data1.fantasiachain.com/",
+    "https://mainnet-data2.fantasiachain.com/",
+    "https://mainnet-data3.fantasiachain.com/"
+  ],
+  "faucets": [],
+  "nativeCurrency": {
+    "name": "FST",
+    "symbol": "FST",
+    "decimals": 18
+  },
+  "infoURL": "https://fantasia.technology/",
+  "shortName": "FSCMainnet",
+  "chainId": 868,
+  "networkId": 868,
+  "explorers": [
+    {
+      "name": "FSCScan",
+      "url": "https://explorer.fantasiachain.com",
```

Comment:
not working

Reply:
It's fixing and re-deploy soon.
Can I submit without explorer and update later?

Reply:
yea - RPCs same - add status incubating

Reply:
Hi @ligi ,
We resolved all functions: RPCs and explorer.
Please recheck and approve our request.
Thanks!